SmartDeviceLinkTests/ProxySpecs/SDLSecondaryTransportManagerSpec.m

```diff
@@ -552,11 +580,34 @@ + (void)swapConnectionMethods {
                     testTransportEventUpdateMessage = [[SDLV2ProtocolMessage alloc] initWithHeader:testTransportEventUpdateHeader andPayload:testTransportEventUpdatePayload.data];
                 });
 
-                it(@"should transition to Connecting state", ^{
+                it(@"should transition to Connecting state but before RAIR is sent", ^{
```

Comment:
Again, this is poor documentation / naming. Are you trying to say / test "it should transition to the connecting state when a transport event update is received and there's no security manager"?